src/java/org/apache/cassandra/tcm/ClusterMetadataService.java

```diff
@@ -125,6 +125,10 @@ public static ClusterMetadataService instance()
 
     private final AtomicBoolean commitsPaused = new AtomicBoolean();
 
+    /**
+     * Returns the state of the {@code ClusteMetadataService}.
```

Comment:
This comment seems to be tautological. Maybe add "returns _current_ state" at least? I would say it could be useful to specify what gets returned, too. Ie whether or not current cluster metadata service is local (ie member of CMS), remote (ie a non-CMS member of the cluster), gossip (ie a node that has not yet been upgraded to TCM), or the node is forced to reset the local metadata.

Reply:
I think that you answer the question that I was wondering. It was unclear to me what the different states means.
How does work the reset of the local metadata?